game/game/inventory.cpp

```diff
@@ -938,6 +938,59 @@ Item *Inventory::findByClass(size_t cls) {
   return nullptr;
   }
 
+Item* Inventory::getNthCategoryItem(enum InvCategory cat, uint32_t num) const {
+  ItmFlags f = ITM_CAT_NONE;
+  switch(cat) {
```

Comment:
We probably shouldn't introduce a new `enum` for a single use-case.
It's probably better to move this translation switch-case to `GameScript::npc_getinvitembyslot`, and use `ItmFlags` here.

Reply:
From what I've read the INV_* values were sort of global constants, but you're right, nothing else is using them.
I'll streamline things a bit, like you suggested.

Reply:
Question: do you prefer num starting at 1 or 0 ? Right now the script "slot" is leaking into the implementation (num = 1 -> first item),